recipe/meta.yaml

```diff
@@ -25,17 +24,19 @@ requirements:
     - python
     - setuptools
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20|0.2.20.*
   run:
     - python
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20|0.2.20.*
```

Comment:
@isuruf let's keep the exact pinning for now but we can move the the `>=` in the next release.